pages/glossary/marked-as-decorative.md

```diff
@@ -13,7 +13,7 @@ The most common ways of marking an element as decorative are to explicitly set `
 
 **Note:** The [semantic role][] of `none` has been introduced in ARIA 1.1 and has less support than the synonym [role][semantic role] of `presentation`. Therefore, authors are encouraged to use `role="none presentation"` to keep the fallback option ([WAI-ARIA description of `presentation` role](https://www.w3.org/TR/wai-aria-1.1/#presentation)).
 
-#### Accessibility Support
+#### Accessibility Support for Definition of Marked as Decorative
```

Comment:
It would be better if you use small letter for word "Definition" and/or "Accessible".
Suggestion is to use one of the following: 
- Accessibility Support for defination of _accessible name_
- Accessibility Support for defination of Accessibile Name